ADR/0025-container-images.md

```diff
@@ -0,0 +1,94 @@
+
+
+### Responsibilities
+
+#### Automated Build and Scanning
+###### Onboard to Pipelines As Code (PaC)
+
+Component Teams are responsible for ensuring their container images are continuously built and scanned for vulnerabilities by following the
+[Extending the Service](https://redhat-appstudio.github.io/infra-deployments/docs/deployment/extending-the-service.html) process to onboard their component to the PaC service. See also [ADR #17](https://github.com/redhat-appstudio/book/blob/main/ADR/0017-use-our-pipelines.md).
+
+***
+#### Container Images <br>
+
+###### Trigger Builds
+Under the PaC service, images are rebuilt when there are updates to the component’s git repository but additional configuration is needed in the Dockerfile to ensure the underlying base (UBI) images are updated with the latest packages (see the [HAS example](https://github.com/redhat-appstudio/application-service/blob/main/Dockerfile#L24])) or at the very least, the latest [security updates](https://developers.redhat.com/articles/2021/11/11/best-practices-building-images-pass-red-hat-container-certification#best_practice__5__include_the_latest_security_updates_in_your_image).   This will minimize the gap between patching and should meet our CVE timelines as long as the repository is active.
+
+###### Scheduled Builds
+
+Since image updates are based on how active our repos are, there is the risk that over time, as code stabilizes and/or enters maintenance mode, the triggers for rebuilds will be less frequent which will cause the images to degrade.  To avoid this, component teams should also ensure there are scheduled builds in place.
+
+
+###### New Components
+Newly onboarded components are required to use a fully supported and patched major version release for their base images per ESS SEC-PATCH-REQ-2 requirement #3.
+
+***
+
+#### Vulnerability Alerts
+
+It is recommended that component teams set up notifications to receive vulnerability alerts that are at least **_medium_** severity.  This can be done in a couple of ways:
+
+* Set up an [alert in quay.io](https://docs.quay.io/guides/notifications.html) which supports email and Slack integration
+* Use the following github action to report vulnerabilities under the action tab.
+You can copy this [workflow](https://github.com/openshift-pipelines/pipeline-service/blob/main/.github/workflows/periodic-scanner-quay.yaml) and this [script](https://github.com/openshift-pipelines/pipeline-service/blob/main/ci/images/vulnerability-scan/scan.sh) onto your repo and set the variables
+
+***
+
+#### Remediation
+
+While our automation process will ensure that component teams are keeping their images updated, security scanners are not perfect.  Vulnerabilities can be reported through other channels, in which case, component teams should assess the severity of these findings and remediate according to the Infosec Remediation Guidelines
```

Comment:
should or must? Do the component teams usually perform the assessment or is Infosec/Prodsec usually helping out with those?

Reply:
It's a *must*.  It's the component teams that assess whether the vulnerability is a risk to their code and take the appropriate action outlined by Infosec